addon-test-support/@ember/test-helpers/-utils.js

```diff
@@ -35,3 +35,14 @@ export function runDestroyablesFor(object, property) {
 
   delete object[property];
 }
+
+/**
+ Returns whether the passed in string consists only of numeric characters.
+
+ @private
+ @param {string} n input string
+ @returns {boolean} whether the input string consists only of numeric characters
+ */
+export function isNumeric(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
Keycodes are only integers. I think you should use parseInt

Reply:
this seems to be the accepted solution for a lot of these questions on StackOverflow with lots of people mentioning that `parseInt()` should not be used for some reason.

ultimately this is only used to catch the case where someone used `'13'` instead of `13`. if someone would put in `'13.0'` in the current state it would also warn, but if we changed it to only allow integers it would no longer show that warning, so I think it is actually better this way